server/scratchy_server/resources/userres.py

```diff
@@ -2,14 +2,21 @@
 from mongoengine import NotUniqueError
 from flask_apispec.views import MethodResource
 from flask_apispec import marshal_with, use_kwargs, doc
+from marshmallow import fields
 
 
 @doc(tags=['User'])
 @marshal_with(UserSchema)
 class UserRes(MethodResource):
 
-    def get(self, userId):
-        return UserModel.objects().get_or_404(id=userId)
+    #@use_kwargs({"pseudo": fields.String()}, location="query")
+    def get(self, userId):#=None, pseudo=None):
```

Comment:
Well can you comment that out ? ;) it should work better

Reply:
done in the previous commit you because you go through yours notif one at a time i think you made the review on the wrong commit